src/logging/NewRelicLoggingService.js

```diff
@@ -17,18 +17,47 @@ function fixErrorLength(error) {
   return error;
 }
 
+/* Constants used as New Relic page action names. */
+const pageActionNameInfo = 'INFO';
+const pageActionNameExpectedError = 'EXPECTED_ERROR';
+
+function logOther(actionName, message, customAttributes) {
+  if (process.env.NODE_ENV === 'development') {
+    console.log(message, customAttributes); // eslint-disable-line
+  }
+  if (window && typeof window.newrelic !== 'undefined') {
+    window.newrelic.addPageAction(actionName, { message, ...customAttributes });
+  }
+}
+
+function logUnexpectedError(error, customAttributes) {
+  if (process.env.NODE_ENV === 'development') {
+    console.error(error, customAttributes); // eslint-disable-line
+  }
+  if (window && typeof window.newrelic !== 'undefined') {
+    // Note: customProperties are not sent.  Presumably High-Security Mode is being used.
```

Comment:
could you explain this more? are customProperties different than customAttributes?